pkg/otlp/config_serverless_noop.go

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+
+// +build serverless
+
+package otlp
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+// IsEnabled no-op
+func IsEnabled(cfg config.Config) bool {
```

Comment:
I would rather add the build tag to the `config_test.go` file than do this

Reply:
I think we can't do that because `IsEnabled` is used here : https://github.com/DataDog/datadog-agent/blob/main/pkg/trace/config/apply.go#L352

Reply:
I am not understanding something here then, is the Trace Agent part of the serverless agent in some way? Otherwise I don't see how this would fail

Reply:
Yes, we use the trace agent in the serverless build, here is the entrypoint : https://github.com/DataDog/datadog-agent/blob/main/pkg/serverless/trace/trace.go#L12